docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
+
+{WFM-RC-NameShort} authentication and authorization module using link:http://passportjs.org[PassportJS].
+This module allows the:
+- Creation and initialization of a Passport authentication service using Passport's local strategy.
+- Protection of express routes from requests by user authentication and authorization.
+- Usage of persistent login sessions using link:https://github.com/expressjs/session[express-session].
+
+
+== Quick Start
+=== Setup
+
+[source,typescript]
+----
+...
+import { PassportAuth, UserRepository, UserService }  from '@raincatcher/auth-passport'
+
+// Initialize user data repository, user service and passport
+const userRepo: UserRepository = new YourUserRepository();
+const userService: UserService = new YourUserService();
+const authService: PassportAuth = new PassportAuth(userRepo, userService);
+...
+authService.init(app, sessionOptions); // Initializes express app to use passport and express-session
+...
+----
+Visit link:https://github.com/expressjs/session[express-session] to find more information about the available express
+session options.
+
+=== Usage
+[source,typescript]
+----
+app.get('/secureEndpoint', authService.protect('admin'), (req: express.Request, res: express.Response) => {
+    res.json({routeName: '/secureEndpoint', msg: 'authenticated and authorized to access secure resource'});
+});
+----
+
+See link:https://github.com/feedhenry-raincatcher/raincatcher-core/tree/master/cloud/passportauth/example[./example] for a sample implementation
```

Comment:
I fully agree with the point being made. "Every reference to github should use some variable I think" - can you elaborate on how you think this would work?